pkgs/top-level/all-packages.nix

```
@@ -533,6 +533,9 @@ with pkgs;
 
   dinghy = with python3Packages; toPythonApplication dinghy;
 
+  djgpp = callPackage ../development/compilers/djgpp { };
+  djgpp_i686 = lowPrio (djgpp.override { i686Build = true; });
+
```

```suggestion
  djgpp = djgpp_i586;
  djgpp_i586 = callPackage ../development/compilers/djgpp { 
    targetArchitecture = "i586";
  };
  djgpp_i686 = lowPrio (callPackage ../development/compilers/djgpp {
    targetArchitecture = "i686"; 
  });

```

Adding both as shown above is useful when someone wants to run both in a controlled, scripted environment.

---

included